pkg/auth/model/model.proto

```diff
@@ -25,8 +25,6 @@ message GroupData {
 // message data model for model.ACL struct
 message ACLData {
     string permission = 1;
-    bool all_repositories = 2;
-    repeated string repositories = 3;
```

Comment:
I wonder if this should be deprecated instead of removing those fields, this way if there will be another feature adding fields to that struct, they won't conflict with potential users which have used this version

Reply:
AFAIU the protobuf is only ever used with an _external_ auth server.  Since no external auth servers support ACLs so far, it seems safe to me to remove.